includes/addon-moderation.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Add a BP Docs-specific "pending" status to use in a moderation workflow.
+ *
+ * @since 2.1.0
+ */
+class BP_Docs_Moderation {
+
+	public $pending_status = 'bp_docs_pending';
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 2.1.0
+	 */
+	public function __construct() {
+	}
+
+	/**
+	 * Hook the Folders functionality into Docs.
+	 *
+	 * @since 2.1.0
+	 */
+	public function add_hooks() {
+		// Register a custom status that's a lot like the built-in "pending" status.
```

Comment:
Why at `9`? I assume this was necessary for some reason.

Reply:
It's been too long for me to remember why this was necessary. I'll remove it and see what happens. :)